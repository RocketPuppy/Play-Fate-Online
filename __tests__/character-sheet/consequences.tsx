import * as React from "react";
import * as renderer from "react-test-renderer";
import Consequences from "../../src/character-sheet/consequences";

test("renders all the pieces", () => {
  const render = renderer.create(<Consequences />);

  expect(render.toJSON()).toMatchSnapshot();
});
