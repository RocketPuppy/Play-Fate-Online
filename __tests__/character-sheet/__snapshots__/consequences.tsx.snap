// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Consequences
  </h2>
  <div>
    <h3>
      Normal
    </h3>
    <div>
      <label>
        <span>
          2
        </span>
        <span>
          Mild
        </span>
        <input
          defaultValue=""
          onChange={[Function]}
          type="text"
        />
      </label>
      <label>
        <span>
          4
        </span>
        <span>
          Moderate
        </span>
        <input
          defaultValue=""
          onChange={[Function]}
          type="text"
        />
      </label>
      <label>
        <span>
          6
        </span>
        <span>
          Severe
        </span>
        <input
          defaultValue=""
          onChange={[Function]}
          type="text"
        />
      </label>
    </div>
    <h3>
      Physical
    </h3>
    <div />
    <button
      onClick={[Function]}
    >
      Add Consequence
    </button>
    <h3>
      Mental
    </h3>
    <div />
    <button
      onClick={[Function]}
    >
      Add Consequence
    </button>
  </div>
</div>
`;
